src/lib/user-ip-address.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (request) => {
+  const forwarded = request.headers["forwarded"];
+  if (forwarded) {
+    const clientIpAddress = forwarded.split(",")[0];
```

Comment:
Can we guarantee the client ip will always be first?
If not, is there an appetite to turn the forwarded header into an object and extract the ip, then rebuild the string.

Reply:
Is there a solution where we use express.headers to turn this string into an object. We can then extract the `for` header and set it maybe